package.json

```diff
@@ -207,6 +208,7 @@
     "tmp": "^0.0.28",
     "tunnel": "0.0.6",
     "underscore": "^1.8.3",
+    "uuid": "^11.0.5",
```

Comment:
I think we already have a util function for UUIDs, presumably to keep dependencies lighter.  If we wanted to pull in a package instead, maybe we can remove that util function.